Sources/FoundationEssentials/FileManager/FileManager+SymbolicLinks.swift

```diff
@@ -59,7 +59,8 @@ extension _FileManagerImpl {
     ) throws {
 #if os(Windows)
         var bIsDirectory = false
-        _ = fileManager.fileExists(atPath: destPath, isDirectory: &bIsDirectory)
+        let absoluteDestPath = URL(filePath: destPath, relativeTo: URL(filePath: path, directoryHint: .notDirectory)).path
```

Comment:
What happens if `path` is absolute?

Reply:
If `path` (the symlink) is absolute, then the behavior doesn't change - the destination is interpreted as relative to that path (so if `path` is `/foo/bar` and `destPath` is `file`, then `absoluteDestPath` is `/foo/file`). If `destPath` (the destination) is absolute, then just the destination path is used (so if `destPath` is `/some/file` then regardless of what `path` is, `absoluteDestPath` is `/some/file`)

Reply:
Yeah I can see if I can form an absolute path based on the CWD for a test case, that's a fair point